api/internal/testutils/validation.go

```diff
@@ -0,0 +1,11 @@
+package testutils
+
+import "testing"
+
+// FailOnError logs the error and terminates the test immediately
+func FailOnError(t *testing.T, err error) {
```

Comment:
The `require` package can be used in place of this. Some of the code in the original repo may need to be updated (out of the scope of this PR).